spec/abci/abci++_methods.md

```diff
@@ -663,6 +664,18 @@ without calling `VerifyVoteExtension` to verify it.
       already passed on to the Application via `PrepareProposalRequest` or `ProcessProposalRequest`.
     * When calling `FinalizeBlock` with a block, the consensus algorithm run by CometBFT guarantees
       that at least one non-byzantine validator has run `ProcessProposal` on that block.
+    * `FinalizeBlockResponse.next_block_delay` - how long CometBFT waits after
+      committing a block, before starting on the new height (this gives the
+      proposer a chance to receive some more precommits, even though it
+      already has +2/3). Set to 0 if you want a proposer to make progress as
+      soon as it has all the precommits. Previously `timeout_commit` in
+      CometBFT config. **Set to constant 1s to preserve the old (v0.34 - v1.0) behavior**.
```

Comment:
> Set to constant 1s to preserve the old (v0.34 - v1.0) behavior

See my comments above about an operator -- oblivious to this -- that wants to upgrade, and has e.g. their `timeout_commit` set to 0 (or to 3).
Also see my comment above about the possibility of using `nil` in this field to allow an app to signal "I'm not aware of this, please fall back to `timeout_commit`"

Reply:
Update: I'm now convinced we don't need to use `nil` here